### PR TITLE
Added Scaffolding for the Style Transfer MPS tests

### PR DESCRIPTION
**Pull request by abhishekpratapa**
Scaffolding for the Style Transfer MPS Tests.

### CODE REVIEW

File: test/neural_net/style_transfer.cxx [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+#define BOOST_TEST_MODULE
+
+#include <boost/test/unit_test.hpp>
+#include <core/util/test_macros.hpp>
+#include <iostream>
+
+#ifdef HAS_MACOS_10_15
+
+struct style_transfer_test {
+public:
```
Comment (shantanuchhabra) [inline test/neural_net/style_transfer.cxx]: "The public:, protected:, and private: keywords should be indented one space."

https://google.github.io/styleguide/cppguide.html#Class_Format

Comment (abhishekpratapa) [reply test/neural_net/style_transfer.cxx]: This is really cool!

File: test/neural_net/style_transfer.cxx [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+#define BOOST_TEST_MODULE
+
+#include <boost/test/unit_test.hpp>
+#include <core/util/test_macros.hpp>
+#include <iostream>
+
+#ifdef HAS_MACOS_10_15
+
+struct style_transfer_test {
```
Comment (nickjong) [inline test/neural_net/style_transfer.cxx]: Why do we need a separate class for this, when you have individual calls to BOOST_AUTO_TEST_CASE below?

File: test/neural_net/style_transfer.cxx [hunk 3]
```diff
@@ -0,0 +1,138 @@
+/* Copyright © 2019 Apple Inc. All rights reserved.
+ *
+ * Use of this source code is governed by a BSD-3-clause license that can
+ * be found in the LICENSE.txt file or at
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+#define BOOST_TEST_MODULE
```
Comment (nickjong) [inline test/neural_net/style_transfer.cxx]: Per the docs, this macro should define the name of the test suite.

https://www.boost.org/doc/libs/1_65_0/libs/test/doc/html/boost_test/tests_organization/test_suite/master_test_suite.html#ref_BOOST_TEST_MODULE

**Pull request merged**